HISTORY.rst

```diff
@@ -1,6 +1,45 @@
 =======
 History
 =======
+3.41.0 (2012-08-04)
+-------------------
+
+Changes
+
+* CCI has a new Metadata ETL task that allows adding fields to existing layouts. Sections can be selected using an index (#2766)
+
+* We added a task to enable an Einstein prediction (thanks, @erikperkins!) (#2778)
+
+* We added standard flows for releasing unlocked packages: release_unlocked_beta and release_unlocked_production (#2768)
+ 
+* We added documentation for using CumulusCI to build managed 2GP packages, 
+  unlocked packages, and extending NPSP and EDA with 2GP packages. (#2768)
+
+* Contributions to CCI now require verification by isort (#2770)
+ 
+* CumulusCI now supports deploying unmanaged dependencies in SFDX source format. (#2735)
+
+* The `create_package_version` task now handles dependencies that use a `zip_url`. (#2735)
+
+* Updates to Github Actions configuration documentation. Thanks @Julian88Tex (#2773)
+
+* CumulusCI now automatically recognizes services and orgs configured via environment variables. See the `docs <https://github.com/SFDO-Tooling/CumulusCI/blob/481ab27446edbfdb038ea62834a6f276dc53bf3d/docs/headless.rst#L9>`_ for more details. (#2676 and #2776)
```

Comment:
I would put this in the Critical Changes section because I think it's pertinent to mention that the EnvironementProjectKeychain is deprecated and should no longer be necessary.

There's also a related change I think is worth calling out, which is that it's no longer necessary to explicitly set CUMULUSCI_KEY on Linux.

We should link to the docs on readthedocs (https://cumulusci.readthedocs.io/en/stable/headless.html), not github!

Reply:
sorry for the bad link

Reply:
Okay, I guess, but please make sure we end up with links to readthedocs both here and in the github release.

Reply:
You don't think we should save "Critical Changes" for changes that are required (failures to be backwards compatible) rather than just recommended?

Reply:
This one is grey area, I'll let you make the call.